src/base/bios/x86/bios.s

```diff
@@ -649,6 +649,26 @@ PKTDRV_driver_entry_ip:
 PKTDRV_driver_entry_cs:
 	.word 0
 
+/* FOSSIL driver signature and jump to original handler */
+	.globl	FOSSIL_isr
+FOSSIL_isr:
+	jmp     1f
+	.globl	FOSSIL_oldisr
+FOSSIL_oldisr:
+	.long   -1
+	.globl	FOSSIL_magic
+FOSSIL_magic:
+	.word   0
```

Comment:
Why not to immediately write the needed
magics here, rather than from fossil.c?

Reply:
Sure we could set magic here, but we still need to set max function number and old isr value from fossil.c, so it seemed neater to keep all the sets in one place.

Reply:
But MAX_FUNC is also a constant, no?
Doesn't matter though.
Thanks!